protos/exocore/core/config.proto

```diff
@@ -33,6 +35,34 @@ message NodeCellConfig {
     }
 }
 
+message NodeStoreConfig {
+    EntityIndexConfig index = 1;
+}
+
+/// Configuration of the entities index
+message EntityIndexConfig {
+    uint64 chain_index_min_depth = 1;
+
+    uint64 chain_index_depth_leeway = 2;
+
+    MutationIndexConfig pending_index_config = 3;
+
+    MutationIndexConfig chain_index_config = 4;
+
+    bool chain_index_in_memory = 5;    
```

Comment:
We shouldn't expose this one as it is for tests and is dangerous since it doesn't persist anything to disk.